src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/bolt/SwitchManagerHub.java

```diff
@@ -27,64 +28,60 @@
 import org.openkilda.messaging.swmanager.request.DeleteLagPortRequest;
 import org.openkilda.persistence.PersistenceManager;
 import org.openkilda.rulemanager.RuleManagerConfig;
 import org.openkilda.rulemanager.RuleManagerImpl;
+import org.openkilda.wfm.error.MessageDispatchException;
 import org.openkilda.wfm.error.PipelineException;
+import org.openkilda.wfm.error.UnexpectedInputException;
 import org.openkilda.wfm.share.flow.resources.FlowResourcesConfig;
 import org.openkilda.wfm.share.hubandspoke.HubBolt;
 import org.openkilda.wfm.share.utils.KeyProvider;
 import org.openkilda.wfm.share.zk.ZkStreams;
 import org.openkilda.wfm.share.zk.ZooKeeperBolt;
 import org.openkilda.wfm.topology.switchmanager.StreamType;
 import org.openkilda.wfm.topology.switchmanager.SwitchManagerTopologyConfig;
+import org.openkilda.wfm.topology.switchmanager.error.SwitchManagerException;
 import org.openkilda.wfm.topology.switchmanager.model.ValidationResult;
 import org.openkilda.wfm.topology.switchmanager.service.CreateLagPortService;
 import org.openkilda.wfm.topology.switchmanager.service.DeleteLagPortService;
 import org.openkilda.wfm.topology.switchmanager.service.LagPortOperationConfig;
 import org.openkilda.wfm.topology.switchmanager.service.SwitchManagerCarrier;
+import org.openkilda.wfm.topology.switchmanager.service.SwitchManagerCarrierCookieDecorator;
+import org.openkilda.wfm.topology.switchmanager.service.SwitchManagerHubService;
 import org.openkilda.wfm.topology.switchmanager.service.SwitchRuleService;
 import org.openkilda.wfm.topology.switchmanager.service.SwitchSyncService;
 import org.openkilda.wfm.topology.switchmanager.service.SwitchValidateService;
-import org.openkilda.wfm.topology.switchmanager.service.impl.SwitchRuleServiceImpl;
 import org.openkilda.wfm.topology.switchmanager.service.impl.ValidationServiceImpl;
-import org.openkilda.wfm.topology.switchmanager.service.impl.fsmhandlers.CreateLagPortServiceImpl;
-import org.openkilda.wfm.topology.switchmanager.service.impl.fsmhandlers.DeleteLagPortServiceImpl;
-import org.openkilda.wfm.topology.switchmanager.service.impl.fsmhandlers.SwitchSyncServiceImpl;
-import org.openkilda.wfm.topology.switchmanager.service.impl.fsmhandlers.SwitchValidateServiceImpl;
 import org.openkilda.wfm.topology.utils.MessageKafkaTranslator;
 
+import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.storm.topology.OutputFieldsDeclarer;
 import org.apache.storm.tuple.Fields;
 import org.apache.storm.tuple.Tuple;
 import org.apache.storm.tuple.Values;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
 @Slf4j
 public class SwitchManagerHub extends HubBolt implements SwitchManagerCarrier {
     public static final String ID = "switch.manager.hub";
 
     public static final String INCOME_STREAM = "switch.manage.command";
 
+    public static final String FIELD_ID_COOKIE = SpeakerWorkerBolt.FIELD_ID_COOKIE;
```

Comment:
Bot must define all data used to define communication streams. It is much easier to open bolt's code and get all required definitions. 

The value itself is copied from other bolt just to avoid creation of one more constant with same value (also it allows reduce number of "names" you will see in storm-ui output).